From cc7f4ce4fa21c107867b8f57f43850b5a86bdb75 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Tue, 29 Apr 2025 15:12:08 +0200
Subject: [PATCH] dt-bindings: soc: stm32: fix stm32mp25-risab example
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix indentation and add necessary clock include.

Change-Id: I20439b9b81259053905525d9426b2cc12994e2c8
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/455192
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 .../bindings/soc/st/st,stm32mp25-risab.yaml      | 16 +++++++++-------
 1 file changed, 9 insertions(+), 7 deletions(-)

--- a/Documentation/devicetree/bindings/soc/st/st,stm32mp25-risab.yaml
+++ b/Documentation/devicetree/bindings/soc/st/st,stm32mp25-risab.yaml
@@ -58,10 +58,12 @@ additionalProperties: false
 
 examples:
   - |
-        risab1: risab@420f0000 {
-          compatible = "st,stm32mp25-risab";
-          reg = <0x420f0000 0x1000>;
-          clocks = <&rcc CK_ICN_LS_MCU>;
-          st,mem-map = <0xa000000 0x20000>;
-          st,srwiad;
-        };
+    #include <dt-bindings/clock/st,stm32mp25-rcc.h>
+
+    risab1: risab@420f0000 {
+      compatible = "st,stm32mp25-risab";
+      reg = <0x420f0000 0x1000>;
+      clocks = <&rcc CK_ICN_LS_MCU>;
+      st,mem-map = <0xa000000 0x20000>;
+      st,srwiad;
+    };
